src/main/java/studio/magemonkey/fusion/data/professions/Profession.java

```diff
@@ -30,11 +30,11 @@ public Profession(long id, UUID uuid, String name, double exp, boolean mastered,
         this.joined = joined;
     }
 
-    public void addExp(long exp) {
+    public void addExp(double exp) {
```

Comment:
Any reason these are now doubles? It feels weird to allow decimals in an experience value.

Reply:
You are right, this was somehow a bad choice from the old code while setting up the api. I have turned it back to long